src/routes/workshop-a11y-fundamentals.svelte

```diff
@@ -203,174 +550,348 @@
-      on the matter.
-    </q>
+  <!-- <article class="t-card t-cta">
+    <h2 class="t-ctaTitle">Join the online workshop!</h2>
+    <p class="t-ctaPitch">Get 9 hours, over 3 days, to refine your A11Y knowledge.</p>
+
+    <dl class="t-ctaPoints">
+      <dt>
+        <svg aria-hidden="true" class="u-svg" style="display: none;">
+          <use xlink:href="#calendar" />
+        </svg>
+        <span class="sr-only">When:</span>
+      </dt>
+      <dd>To be defined (April?)</dd>
+
+      <dt>
+        <svg aria-hidden="true" class="u-svg" style="display: none;">
+          <use xlink:href="#clock" />
+        </svg>
+        <span class="sr-only">Hour:</span>
+      </dt>
+      <dd>{eventHour} <span class="tz">{eventTZ}</span></dd>
+    </dl>
+
+    <a href="" class="t-ctaBtn">Buy ticket for {price}</a>
+    <span class="t-ctaNote">Price is Early Bird</span>
+
+  </article> -->
+  <WorkshopForm formEndpoint={endpointA11Y}>
+    <p class="t-ctaPitch">Get 9 hours, over 3 days, to refine your A11Y knowledge.</p>
+
+    <dl class="t-ctaPoints">
+      <dt class="time-todo">
+        <svg aria-hidden="true" class="u-svg" style="display: none;">
+          <use xlink:href="#calendar" />
+        </svg>
+        <span class="sr-only">When:</span>
+      </dt>
+      <dd>Soon</dd>
+
+      <dt>
+        <svg aria-hidden="true" class="u-svg" style="display: none;">
+          <use xlink:href="#clock" />
+        </svg>
+        <span class="sr-only">Hour:</span>
+      </dt>
+      <dd>{eventHour} <span class="tz">{eventTZ}</span></dd>
+    </dl>
+  </WorkshopForm>
+
+
+
+  <h2 class="t-title asH2">
+    F.A.Q.
+  </h2>
+
+  <div class="faqArea">
+    <ul class="faqList">
+    <li>
+        <Accordion summary="What does A11Y mean?">
+          A11Y stands for 'accessibility'.
+          The irony about the word "accessibility" itself is that it's not very easy to pronounce.
+          So, it was created this abbreviation: A + 11 characters + Y, which has a more friendly homophone of "ally".
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="Will the workshop be recorded?">
+          No, it won't be recorded, but you’ll have access to all the materials shared.
+        </Accordion>
+      </li>
+
+
+      <li>
+        <Accordion summary="What materials are provided?">
+         <ul class="t-list">
+          <li>
+            The materials slides (+ 50 slides);
+          </li>
+          <li>
+            Codebase with all the mentioned resources;
+          </li>
+          <li>
+            Exercises, including the solutions explained;
+          </li>
+         </ul>       <!-- TODO list -->
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="I know A11Y. Will I learn something new?">
+          Oh, yeah. Even me, every time I give this workshop I always learn something new. Unless you are an A11Y expert, I’m confident that you'll fill some gap.
```

Comment:
Oh, yeah. Personally, every time I give this workshop I always learn something new. Unless you are an A11Y expert, I’m confident that you'll fill some gaps.